src/VisualStudio/Core/Def/Implementation/Venus/ContainedDocument.cs

```diff
@@ -62,6 +62,13 @@ internal sealed partial class ContainedDocument : ForegroundThreadAffinitizedObj
 
         public static ContainedDocument TryGetContainedDocument(DocumentId id)
         {
+            // Liveshare creates their own non-VS workspace which downloads files from the host.
+            // In such a case, a null id can be passed in, so check before attempting to retrieve the document.
```

Comment:
This comment seems out of place to me. It feels like either:

1. The behavior makes sense for this method, and doesn't need further justification, or,
2. Really this is a terrible hack and maybe it shouldn't be here.

This feels like fixing a bug in int.Parse and then commenting that "it's needed for JSON parsing". This is a core bit of code -- it shouldn't be thinking about very specific clients or something seems off.

Reply:
(I'd go with 1 and say just remove the comment.)

Reply:
Sure I can remove it - checking with @heejaechang are you fine without the comment?